pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -40,7 +40,7 @@ protected void internalInit(final WebContext context) {
     }
 
     @Override
-    public final HttpAction redirect(final WebContext context) throws HttpAction {
+    public HttpAction redirect(final WebContext context) throws HttpAction {
```

Comment:
Good. Is this PR still necessary? Please close it if you don't need this change anymore.